WordPress/src/main/res/layout/home_page_picker_bottom_toolbar.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/bottomToolbar"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/mlp_bottom_shadow_height"
+        android:background="@drawable/modal_layout_picker_bottom_shadow" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="?attr/categoriesBackground"
+        android:gravity="center"
+        android:orientation="horizontal"
+        android:paddingBottom="@dimen/mlp_bottom_button_vertical_margin"
```

Comment:
Same comment as before: Do you think these dimensions generic enough now to rename them?

Reply:
As above fixed with 963619e2ac8f22eaa9fc8efb273f61ab5633c3bb